From 8405b621ecd0ce8b8f97574afe8ea866f68c9371 Mon Sep 17 00:00:00 2001
From: Helmut Buchsbaum <helmut.buchsbaum@opensource.tttech-industrial.com>
Date: Tue, 7 Apr 2020 11:16:38 +0200
Subject: dm: Do not link to zlib

Since zlib is not used, remove linking to it. This reduces build
dependencies.

Tracked-On: #4838
Signed-off-by: Helmut Buchsbaum <helmut.buchsbaum@opensource.tttech-industrial.com>
---
 devicemodel/Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/devicemodel/Makefile b/devicemodel/Makefile
index 7213642a..4b12c6c7 100644
--- a/devicemodel/Makefile
+++ b/devicemodel/Makefile
@@ -66,7 +66,6 @@ LIBS = -lrt
 LIBS += -lpthread
 LIBS += -lcrypto
 LIBS += -lpciaccess
-LIBS += -lz
 LIBS += -luuid
 LIBS += -lusb-1.0
 LIBS += -lacrn-mngr
-- 
2.20.1

